app/components/publications/publication/case/organization-add-modal.hbs

```diff
@@ -0,0 +1,44 @@
+<Auk::Modal
+  @size="medium"
+  data-test-add-organisationModal>
+  <Auk::Modal::Header
+    @title={{t "add-organization"}}
+    @onClose={{@onCancel}} />
+  <Auk::Modal::Body>
+    <div class="auk-form-group-layout auk-form-group-layout--standard">
+      <div class="auk-form-group {{this.validators.name.errorClass}}">
+        <Auk::Label for="organizationNameInput"
+          @required={{true}}>
+          {{t "name"}}
+        </Auk::Label>
+        <Auk::Input type="text" id="organizationNameInput"
+          @block={{true}}
+          @value={{this.name}}
+          {{on "input" this.validators.name.enableError}} />
+      </div>
+      <div class="auk-form-group">
+        <Auk::Label for="organizationIdentifierInput">
+          {{t "identifier"}}
```

Comment:
Aan dit label zou ik toevoegen wat we verwachten als identificator om de gebruiker te helpen. Dus zoiets als  `Identificator (OVO-nummer)`

@ValenberghsSven Sorry als je dit vroeger al eens gehint had. Dan heb ik dat toen verkeerd begrepen.